README.md

```diff
@@ -120,6 +121,16 @@ For this suite the `runner.json` file looks like this:
 The tests use a configuration file `/config/vectors.json` to configure test vectors.
 [Test Vector configuration is documented in testVectorGuide.md,](/testVectorGuide.md)
 
+### Running Interoperability Tests
+Running interoperability tests requires having authorization to multiple implementations'
+endpoints. Because most users of this suite will not have those authorization capabilities
+the interoperability suites are disabled by default. If you wish to try running the interoperability suites
+you may by setting `local: false` in `./config/runner.json` or using the ENV Variable `LOCAL_ONLY=false`.
+
+```
```

Comment:
```suggestion
```bash
```

Reply:
You can also just use `sh` to get the same result, fwiw.

Reply:
Yeah totally. I noticed we’re already using bash elsewhere, hence this suggestion. I typically use sh